frontstage/templates/passwords/forgot-password.check-email.html

```diff
@@ -20,7 +20,7 @@ <h1 class="saturn">Password reset request sent</h1>
               <li>Check your spam folder</li>
               <li>Check that you've entered your email address correctly </li>
               <li>Ensure that you've registered an account using this email address </li>
-              <br>If you're still having problems, call us on 0300 1234 931 or send an <br/>email to <a href="mailto:surveys.ons@gov.uk">surveys.ons@gov.uk</a>.</p>
+              <br>If you're still having problems, call us on 0300 1234 931 or send an <br/>email to <a href="mailto:surveys@.ons.gov.uk">surveys@ons.gov.uk</a>.</p>
```

Comment:
you've got a `.` after the `@` you can remove

Reply:
All done :+1: